build.sh

```diff
@@ -41,3 +41,6 @@ npx tsc ./src/index.mjs \
   --allowJs \
   --emitDeclarationOnly \
   --outDir $outdir/types
+
+mv $outdir/types/opa.d.ts $outdir/types/opa.d.mts
+cp $outdir/types/opa.d.mts $outdir/types/opa.d.cts
```

Comment:
Unfortunately separate declaration files are required. Otherwise, typescript will assume the types are either ESM or CJS depending on the value of the `type` field of the nearest package.json.